cryptofeed/exchange/bitmex.py

```diff
@@ -473,6 +477,11 @@ async def _instrument(self, msg: dict, timestamp: float):
         }
         """
         for data in msg['data']:
+            if 'indicativeFundingRate' in data:
```

Comment:
So the predicted next funding rate is only provided in the open interest callback? Such an annoying API! Makes sense to do this.

Reply:
yeah it truly is awful :(